### ISSUE TITLE
Merchant Re-Install Major Problem

### ISSUE THREAD

**Issue by resistorsoftware** [Shopify/shopify_app#1412]
This has been happening for a few months now. A lot of merchants are trying to re-install this App, but this is what Shopify is showing, based on them clicking the Add App button from App Store. My question is, what causes this at Shopify. So far, no one at Shopify has been able to explain the situation that causes this. The Shopify App version for this is currently 18.1.1 with API version 9.5.1 

Shopify installs, uninstalls, and re-installs with no problem, but merchants cannot do that. WHY?????

<img width="723" alt="unsupportedApp" src="https://user-images.githubusercontent.com/123307/163264714-5e67a967-d02b-4855-8fc1-0e9b705baac2.png">

**Comment by resistorsoftware** [comment]
I am so very curious as to the cause of this issue. My logs show no hits from merchants attempting installs, so it is as if Shopify is intercepting the App Store requests and somehow flashing this message, but only to merchants that are not me or Shopify.

**Comment by resistorsoftware** [comment]
No. The App is clean as a whistle in terms of API notifications. It is a Rails 7, Shopify App 18 series App using 2022-01 API calls for everything. So what I see in the App, are merchants connecting, and then at the oAuth phase of things, Shopify is somehow pooched and throwing up this banner about the App being "uninstallable".

Obviously Harley has a switch in his office and he is not pleased with me... I need that switch turn off again.

**Comment by gonzaloriestra** [comment]
@resistorsoftware it looks like it had been blocked for using a deprecated API version (2020-10), but it should be working now. I'll try to gather more information about it.

So I'm closing here, but if you notice it again, please re-open or let me know.

Sorry again for the inconvenience!

**Comment by resistorsoftware** [comment]
Yes. This was indeed the problem. I am very pissed off at Shopify for this amateur hour behavior, as it reflects very poorly on both the amount of effort employees made to fix a problem of their own making, and also, that apparently the tools in use to monitor Apps by Shopify are also so disconnected that these kinds of problems happen. 

All in all, very regretful I placed my good faith in being served well by Shopify in this case. Zero out of five stars for taking 6 months to diagnose and fix a basic issue internal to Shopify, especially when repeatedly provided with facts that there was indeed a strange install situation happening. I took Shopify at their word and at face value and accepted the advice given that there was no internal issue, and that cost me 6 months of installs.
